src/common/transformations/src/transformations/common_optimizations/convert_pagedattn_inputs.cpp

```diff
@@ -140,6 +140,9 @@ ov::pass::ConvertPagedAttnInputs::ConvertPagedAttnInputs(const KVCacheConfig& co
 
         key_cache->set_partial_shape(key_cache_shape);
         value_cache->set_partial_shape(value_cache_shape);
+
+        key_cache->validate_and_infer_types();
```

Comment:
@dmitry-gorokhov @ilya-lavrenov  I find out that after setting the shape and precision of `Parameter` node, it's important to call `validate_and_infer_types`, otherwise the `output` of Parameter is not updated to correct value, therefore it triggers the error in `CommonOptimizations`